src/context_store/orchestrator.py

```
@@ -338,7 +342,12 @@ async def start_lifecycle(self) -> None:
 
     async def dispose(self) -> None:
         """全アダプターのリソースを解放する。"""
+        # まずライフサイクルマネージャーを終了させ、タスクの完了を待機する
         await self._lifecycle_manager.graceful_shutdown()
+
+        # 残っているバックグラウンドタスクがあればキャンセル（5s タイムアウト）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)